backend/middlewares/validators/util.ts

```diff
@@ -39,6 +39,26 @@ export const validateArray = (value: any, type: Type): boolean => {
   );
 };
 
+export const validateObject = (
+  object: { [field: string]: any },
+  types: { [field: string]: any },
+): boolean => {
+  return (
+    object !== undefined &&
+    object !== null &&
```

Comment:
If I'm remembering my falsy values right, this can all become `return (object) && ...`